apps/web/README.md

```diff
@@ -1,30 +1,189 @@
+7. Next, you need to paste the `did.json` in the /.well-known/ directory. Save it to the `/public/.well-known/` directory. Be sure to replace the existing did.json file.
+
+   - Download `did.json` (Step 2: “Download did:web”) and place it at `/public/.well-known/did.json`
+
+8. Once the new files are saved, on the APIs tab in Cloud, find the toggle switch next to the Notify API section and switch it on. You should see a success toast: "Notify configuration successfully verified"
+
+### Test and view notifications
+
+Now you should have a fully functioning dapp that is capable of sending notifications with the Web3Inbox SDK.
+
+First, test that you can subscribe to notifications by going to your dapp's public URL, connect your wallet, sign the SIWE message in your wallet, and press subscribe. Approximately every 12 seconds you should see a "New block" notification in the dapp's UI.
+
+To quality for bounties, notifications must be visible [app.web3inbox.com](https://app.web3inbox.com) or one of our sample wallets that supports Web3Inbox notifications. Note that these automated notifications will only be sent to your account if you have the UI open and with the same account connected.
+
+Sample wallets to test notifications:
+
+- [iOS](https://testflight.apple.com/join/09bTAryp)
+- [Android](https://appdistribution.firebase.google.com/testerapps/1:1056012215045:android:1c076c984136751211cd6c/releases/410ohp8retc9o?utm_source=firebase-console)
+
+### Sending notifications
+
+1.  Send test notification button in the gm-hackers dapp.
+2.  Use [WalletConnect Cloud](https://cloud.walletconnect.com) broadcast feature.
+3.  Make a request to Notify API. The example below showcases how to send a notification to all subscribers.
+
+    ```js
+    // Your project ID from WalletConnect Cloud
+    const projectId = process.env.NEXT_PUBLIC_PROJECT_ID;
+    // notify_api_secret generated in WalletConnect Cloud
+    const notifyApiSecret = process.env.NOTIFY_API_SECRET;
+
+    const headers = {
+      Authorization: `Bearer ${notifyApiSecret}`,
+    };
+
+    // 1. Get the list of subscribers for your project
+    const subscribersRes = await fetch(
+      `https://notify.walletconnect.com/${projectId}/subscribers`,
+      { headers }
+    );
+    const subscribers = await subscribersRes.json();
 
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+    // 2. Send a notification to all your subscribers
+    const body = JSON.stringify({
+      accounts: subscribers,
+      notification: {
+        title: "GM Hackers!",
+        body: "Enjoy hacking on WalletConnect",
+        icon: "https://avatars.githubusercontent.com/u/37784886?s=48&v=4",
+        url: "https://hackers.gm.walletconnect.com/",
+        type: "promotional",
```

Comment:
yep. this should be updated in gm hackers repo too. docs are much more fleshed out now so not a big deal, but only following up because this change at the time had me stuck for a while wondering why notifications weren't being sent, all of a sudden since at the time i was referencing GM hackers.